partner_tz/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Partner timezone",
+    "summary": "Remove partner timezone default value and display on form",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Uncategorized",
```

Comment:
May be the category can be improve

Reply:
Do you have a suggestion? It's only depending on base, and looking at other modules in this repo, I'm not sure there are much better fits :shrug: